src/org/ods/services/SonarQubeService.groovy

```diff
@@ -48,16 +52,18 @@ class SonarQubeService {
         }
     }
 
-    def generateCNESReport(String projectKey, String author) {
+    def generateCNESReport(String projectKey, String author, String sonarBranch, String sonarQubeEdition) {
         withSonarServerConfig { hostUrl, authToken ->
+            def branchParam = sonarQubeEdition == 'community' ? '' : "-b $sonarBranch"
```

Comment:
Doesn't this need to be `"-b ${sonarBranch}"`? At least for consistency we should do that ...

Reply:
For a single param this is enough but I changed it nevertheless now for consistency. Btw. I saw other cases where we also do it that way, e.g. https://github.com/opendevstack/ods-jenkins-shared-library/blob/master/src/org/ods/component/Pipeline.groovy#L308